packages/mock-app/src/pages/Verify.tsx

```diff
@@ -89,7 +89,7 @@ const Verify = () => {
     setErrorMessages(errorMessages);
     setCurrentScreen(screen);
   };
-
+  // TODO: Move this function to the vckit service
```

Comment:
Can you explain more about this comment?

Reply:
Currently, the `issueVC` and `issueCredentialStatus` functions are located in the vckit service (packages/services/src/vckit.service.ts). I think it would make sense to move the `verifyCredential` function there as well, so everything is in one place. This would make it easier to manage and allow us to reuse the `verifyCredential` function more effectively.